client/my-sites/hosting/sftp-card/ssh-keys.scss

```diff
@@ -2,11 +2,12 @@
 	max-width: calc(100vw - 32px);
 }
 
-.hosting__main-layout-col .card.ssh-keys-card {
-	padding-left: 16px;
-	padding-right: 8px;
-	display: flex;
-	gap: 8px;
+@include breakpoint-deprecated( ">1280px" ) {
```

Comment:
We ended up with both the CSS-in-JS version and the plain CSS version 🤣 Let me know if this is a mistake @danielbachhuber, I can put up a PR to remove it!

Reply:
@zaguiini I [didn't end up using](https://github.com/Automattic/wp-calypso/pull/69996#discussion_r1022040820) CSS objects. I think it might've been left in @mpkelly's refactor, which maybe means the `&&` trick doesn't actually work...

Reply:
This one's on me 🤦🏻. I removed the CSS and then found I needed triple ampersands to have the rule take effect. 

FYI the ampersand is just a reference to the current class, so `&&&` is like `.my-class.my-class.my-class`. 

PR created https://github.com/Automattic/wp-calypso/pull/70041